packages/core/src/input/enums/KeyCode.ts

```diff
@@ -0,0 +1,424 @@
+/**
+ * The number of the code in the keyboard event.
```

Comment:
Don't need to express concepts of 'number, for example `Defines the keys on a keyboard.` is enough.